packages/ag-charts-enterprise/src/components/popover/popover.ts

```diff
@@ -0,0 +1,85 @@
+import { _ModuleSupport, type _Util } from 'ag-charts-community';
+
+const { BaseModuleInstance, createElement } = _ModuleSupport;
+const canvasOverlay = 'canvas-overlay';
+
+export interface PopoverOptions {
+    ariaLabel?: string;
+    class?: string | Array<string>;
+    role?: string;
+    onClose?: () => void;
+}
+
+/**
+ * A non-modal element that overlays the chart.
+ */
+export class Popover<Options extends PopoverOptions = PopoverOptions>
+    extends BaseModuleInstance
+    implements _ModuleSupport.ModuleInstance
+{
+    private readonly moduleId: string;
+    private readonly element: HTMLElement;
+    private content?: Array<HTMLElement>;
+
+    constructor(
+        protected readonly ctx: _ModuleSupport.ModuleContext,
+        id: string
+    ) {
+        super();
+
+        this.moduleId = `popover-${id}`;
+
+        this.element = ctx.domManager.addChild(canvasOverlay, this.moduleId);
+        this.element.role = 'presentation';
+
+        this.destroyFns.push(() => ctx.domManager.removeChild(canvasOverlay, this.moduleId));
+    }
+
+    public show(options: Options) {
+        const popover = createElement('div');
```

Comment:
Maybe we could return this so things like the colour picker can attach css custom properties directly?